api/src/main/java/ai/djl/Application.java

```diff
@@ -181,6 +181,9 @@ public int hashCode() {
          * String}&gt;.
          */
         Application WORD_RECOGNITION = new Application("cv/word_recognition");
+
+        /** An application that accepts a seed and returns generated images. */
+        Application GAN = new Application("cv/gan");
```

Comment:
Maybe use Generator instead. A GAN is a technique and an Application is supposed to be a problem

Reply:
How about "GENERATION" to keep with the standard in Application